gk/main.c

```diff
@@ -2254,7 +2257,7 @@ gk_proc(void *arg)
 
 		process_cmds_from_mailbox(instance, gk_conf);
 
-		if (fe != NULL) {
+		if (fe != NULL && fe->in_use) {
```

Comment:
We need to test `rte_rdtsc() >= fe->expire_at` as well because the entry may have been renewed by a message.